feature: add last_used_at to public_keys

macno opened the pull request.

macno committed:
deps: update babel+eslint+prettier

macno committed:
run lint:fix

macno committed:
Dockerfile: update base image to node:14

macno committed:
tests: testRestKey: test last_used_key

gizero reviewed (commented):
Beyond comments below, and a small typo in commit message for aefcc5f: `s/prittier/prettier/`, LGTM!

macno committed:
KeyManager: call correct function

this was not a bug: dbClient.[update,delete] call the same
underlying function query

macno committed:
public keys: add last_used_at column

if authorized_keys endpoint receives a valid fingerprint,
updates the public_key record to store the current timestamp in last_used_at column

macno commented on src/lib/managers/KeyManager.js:
```diff
@@ -54,7 +54,7 @@ class KeyManager {
   async update(account_id, id, ssh_options = '') {
     const sql = 'update public_keys set key_ssh_options = ? where id = ? and account_id = ?';
-    const changes = await this.db.delete(sql, [ssh_options, id, account_id]);
+    const changes = await this.db.update(sql, [ssh_options, id, account_id]);
```
Yes, indeed a "while-at-it-fix".   
I moved it in a separate commit

macno commented:
> small typo in commit message for [aefcc5f](https://github.com/theoapp/theo-node/commit/aefcc5f27c327690d4b06ddddeb6b2144241f57d): `s/prittier/prettier/`, LGTM!

fixed
